src/utils/minio.ts

```diff
@@ -453,14 +453,14 @@ export const removeObjectsFromBucket = async (
   const objects = await getObjectList(bucketName, prefix);
   if (backendStorage === "backblaze") {
     await Promise.all(
-      objects.map((o) => removeObjectFromStorage(bucketName, o.name))
+      objects.map((o) => removeObjectFromStorage(bucketName, o.name!))
```

Comment:
I opted to leave the filters out and let the assertion fail in hopes it helps log any issues. Also on line 463.